index.html

```diff
@@ -8,30 +8,25 @@
 </head>
 <body>
     
-<header>
+<header class="display-out">
 <h1>Stefan Korder</h1>
-<div>
 <nav>
     <a href="#contact"><button type="submit">Kontakt</button></a>
```

Comment:
Submit buttons are usually only used in forms to submit data to a server (or similar). 
Using the anchor tag and styling that should be sufficient here :)